htdocs/public/ticket/create_ticket.php

```diff
@@ -307,7 +307,7 @@
 			//setEventMessages($langs->trans('YourTicketSuccessfullySaved'), null, 'mesgs');
 
 			// Make a redirect to avoid to have ticket submitted twice if we make back
-			setEventMessages($langs->trans('MesgInfosPublicTicketCreatedWithTrackId', '<strong>'.$object->track_id.'</strong>', '<strong>'.$object->ref.'</strong>'), null, 'warnings');
+			setEventMessages($langs->transnoentities('MesgInfosPublicTicketCreatedWithTrackId', '<strong>'.$object->track_id.'</strong>', '<strong>'.$object->ref.'</strong>'), null, 'warnings');
```

Comment:
The ->tans should be required because setEventMessages will then output the string with ->trans only if string is a translation keyword. If a full text, it will remains.
Did you trouble was on a char inside the trackid or the ref ?

Reply:
It display "<strong>.... </strong>" Into the set event message. Maybe we should let trans but put html into translation files?

Reply:
No we must avoid html into translation as it is "presentation related" task.
I am still not able to reproduce your bug. Can you paste a screenshot ?

Reply:
Does this fix 5a806c1ece7d9fa59a9bdc75264ea9bbab63a0f7 is ok for you ?